client/src/components/Reviews.jsx

```diff
@@ -1,13 +1,9 @@
-var Reviews = ({reviewsData}) => {
-	return (
-		<div>
-			{reviewsData}
-		</div>
-	)
-}
+var Reviews = ({reviewsData}) => (
+	<div>
+		{reviewsData.map((review, i) => 
```

Comment:
@gorakong Do you need the index of a specific review for this map? If not, then there isn't really a reason to explicitly pass it into your callback.

Reply:
@jcelder It is indeed unnecessary! I put it in originally with the intent of using it as a key prop for each review object.